### ISSUE TITLE
Backup still not working in latest version 3.2.5

### ISSUE THREAD

**Issue by rudilaskowicz** [crobertsbmw/MyTargets#13]
On my OnePlus 7T pro running Android 12 the backup still isn't working, neither the local or cloud, of I could just get to the local backup location I could manually upload to Google drive but even that no longer works.
The app is great but the fact the backup doesn't work is a real disappointment, last time I updated my phone os I lost over a year of scores, please can this be fixed. Many thanks for your brilliant app, once the backup issue is resolved it will be perfect.
Kind regards
Rudi

**Comment by jatkinson1000** [comment]
I assume the latest update on Google Play fixes this issue @guyandtheworld ?

**Comment by guyandtheworld** [comment]
Yes @jatkinson1000, the latest update fixes this. Closing this issue.
